common/containers/Tabs/Contracts/components/Interact/components/InteractExplorer/components/Fields.tsx

```diff
@@ -12,9 +11,8 @@ export class Fields extends Component<OwnProps> {
   public render() {
     const makeContent = () => (
       <React.Fragment>
-        <GasLimitField />
+        <GasSlider />
```

Comment:
We're going to need a `hideData={true}` prop, since on contract interact, the data is determined from the rest of the form. We don't want a user entering data here.